app/views/languages/language.erb

```diff
@@ -48,7 +55,38 @@
 
     <div class="col-md-8 tab-content">
 
-      <div class="active tab-pane" id="about">
+      <% if !track.active %>
+        <div class="active tab-pane" id="launch">
+          <h5>Help Us Launch the <%= track.language %> Track!</h5>
+
+          <p>
+            Launching a track takes a lot of work and we'd love your help!
+          </p>
+
+          <h5> In order to launch we need: </h5>
+
+          <p>
+            <ul>
+              <li>A submodule in x-api</li>
```

Comment:
I just now decided that we don't have to say this, because I'll add it as a submodule when I create the repo. I've added it to my checklist.